dts/bindings/arm/nordic,nrf-egu.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
Could shorten the header a bit to

```yaml
# Copyright (c) 2019 Nordic Semiconductor ASA
# SPDX-License-Identifier: Apache-2.0
```

I changed it to that in a bunch of other files.